database/migrations/2022_11_11_065414_add_on_trainer_report_to_position.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up(): void
+    {
+        Schema::table('position', function (Blueprint $table) {
+            $table->boolean('on_trainer_report')->default(false)->nullable(true)->after('on_sl_report');
```

Comment:
Set nullable to false. Any boolean column should be either true or false and not include a third option, null.

Reply:
works for me, done. I copied that from position.on_sl_report, which is nullable